spec/json_matchers/match_response_schema_spec.rb

```diff
@@ -113,4 +113,21 @@
     expect(valid_response).to match_response_schema("collection")
     expect(invalid_response).not_to match_response_schema("collection")
   end
+
+  it "works when a pure string is used for response" do
+    create_schema("foo_schema", {
+      "type" => "object",
+      "required" => [
+        "id",
+      ],
+      "properties" => {
+        "id" => { "type" => "number" },
+        "title" => {"type" => "string"},
+      },
+      "additionalProperties" => false,
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.<br>Put a comma after the last parameter of a multiline method call.